src/mono/mono/component/mini-wasi-debugger.c

```diff
@@ -0,0 +1,172 @@
+#include <glib.h>
+#include <mono/mini/debugger-agent-external.h>
+#include <mono/component/debugger-engine.h>
+#include <mono/metadata/components.h>
+
+#ifdef HOST_WASI
+static int conn_fd;
+static int log_level = 1;
+static int retry_receive_message = 100;
+static int connection_wait = 1000;
+
+__attribute__((import_module("wasi_snapshot_preview1")))
```

Comment:
It might never need to change, or if it does, it will be along time from now when the WASI spec evolves significantly. Since this only gets declared once, I don't think we benefit from moving the definition somewhere else - that would just make it harder to see what's going on.